docs/developer/stock-allocation.mdx

```diff
@@ -41,13 +43,51 @@ classDiagram
     class Warehouse{
       click_and_collect_option: String
     }
+    class Channel{
+      allocationStrategy: String
+    }
 ```
 
 ## Allocation flow
 
-Stocks are allocated in the checkout completion process for all lines
-with track inventory turned on. The allocation is made for every line
-on stocks with the highest quantity available value. Multiple allocations are created when there is insufficient available quantity in one stock. Insufficient quantity in all product stocks raises the `InsufficientStock` error.
+Stocks are allocated in the checkout completion process, for all lines
+that have track inventory turned on. The allocation is made for every line
+according to the [allocation strategy](#allocation-strategies) in the given channel.
+Multiple allocations are created when there is insufficient available quantity in one stock.
+Insufficient quantity in all product stocks raises the `InsufficientStock` error.
+
+The line can be allocated only in the stocks that fulfill the following conditions:
+
+- the stock's warehouse is available in the channel in which the order is created
+- the stock's warehouse has a shipping zone that includes the shipping address
+  (or billing address if the shipping address is not set) country code
+  and this shipping zone is available in the channel in which the order is created
+- the stock has available quantities
+
+### Allocation strategies
+
+The allocation strategies allow defining the rule that is used during the stock
+allocation. At this moment two possible options are available:
+
+- Prioritize warehouses by sorting order (`PRIORITIZE_SORTING_ORDER`) - allocate stocks
+  according to the warehouses' order within the channel. If stock is insufficient,
+  the remaining quantity is allocated in the next warehouse on the list
+  and repeated if necessary.
+  To change the warehouses' order within the channel, use
+  [`channelReorderWarehouses`](developer/api-reference/mutations/channel-reorder-warehouses.mdx) mutation.
+- Prioritize warehouses with highest stock (`PRIORITIZE_HIGH_STOCK`) - allocate stock
+  in a warehouse with the most stock. If not enough stock is available in a single
+  warehouse, the remaining quantity is allocated in the next warehouse on the list
+  and continues to the next one if necessary.
+  In this scenario, the warehouses' order within the channel is ignored.
+
+## Allocation flow
```

Comment:
Why do I see it twice? Line 51 is the same...

Reply:
I didn't notice that during rebasing.